src/components/create/SendBitcoin.vue

```diff
@@ -235,21 +227,17 @@ export default class SendBitcoinLedger extends Vue {
     this.currentComponent = 'PegInForm';
     this.txId = '';
     this.txError = '';
-    this.createdTx = {
-      coin: EnvironmentAccessorService.getEnvironmentVariables().vueAppCoin,
-      inputs: [],
-      outputs: [],
-    };
-    this.amount = new SatoshiBig('0', 'satoshi');
-    this.refundAddress = '';
-    this.recipient = '';
-    this.feeBTC = new SatoshiBig('0', 'satoshi');
-    this.txBuilder = new LedgerTxBuilder();
+    this.setTxBuilder();
+    await this.walletService.stopAskingForBalance();
```

Comment:
Should we have this as a store action instead?

Reply:
yes and the TxBuilder should be a getter on the next PR